src/libstd/time/duration.rs

```diff
@@ -45,6 +45,7 @@ macro_rules! try_opt {
 
 /// ISO 8601 time duration with nanosecond precision.
 /// This also allows for the negative duration; see individual methods for details.
+#[unstable(feature = "std_misc")]
```

Comment:
It was only HashState that I found myself unable to use; I didn’t try any of the rest. But still, they should have these markers, should they not?